Makefile.PL

```diff
@@ -3,10 +3,15 @@ use strict;
 use Getopt::Long qw(GetOptions);
 use ExtUtils::MakeMaker;
 
+my $i;
+my $tk_patchLevel;
+
 sub _die ($) {
     # CPAN smokers report FAIL if Makefile.PL dies, it should exit with status 0
     my $err = shift;
     warn $err;
+    # Cleanly exit Tcl/Tk if it was loaded
+    $i->Eval('destroy .') if ($tk_patchLevel);
```

Comment:
To avoid issues related to #14, if Tk is loaded then the root Tk window is destroyed before Makefile.PL exits.